community-members.md

```diff
@@ -13,7 +13,7 @@ in alphabetical order:
 - [Morgan McLean](https://github.com/mtwo), Splunk, until October 2026
 - [Pablo Baeyens Fernandez](https://github.com/mx-psi), Datadog, until October 2026
 - [Severin Neumann](https://github.com/svrnm), Independent, until October 2025
-- [Ted Young](https://github.com/tedsuo), Lightstep, until October 2025
+- [Ted Young](https://github.com/tedsuo), Grafana, until October 2025
```

Comment:
Make sure to also update https://github.com/open-telemetry/community/blob/main/areas-of-interest.md#ted-young-lightstep